tensorflow/python/ops/data_flow_ops.py

```diff
@@ -920,6 +920,74 @@ def __init__(self,
     super(PaddingFIFOQueue, self).__init__(dtypes, shapes, names, queue_ref)
 
 
+class GPUCompatiblePaddingFIFOQueue(QueueBase):
```

Comment:
Why do we need a separate class here? Why not just add GPU support to PaddingFIFOQueue?

Reply:
Did not implement enqueue_many/dequeue_many. This would not affect the original PaddingFIFOQueue.